examples/autoconf/build.gradle

```diff
@@ -19,9 +19,11 @@ plugins {
 	id 'com.google.cloud.tools.jib'
 }
 
-
 description = 'Examples for using java autoconfiguration and Google Cloud Operations'
 
+// examples are not published, so version can be hardcoded
+version = '1.0.0'
```

Comment:
can we use 0.1.0 to ensure these aren't be seen as stable, even if they aren't published?